[orc8r]: API: mconfig; sgi interface static IP address config.

pshelar opened the pull request:
## Summary

Following API would allow operator configure a static
IP address for SGI management interface.

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

<!--
    Tag your PR title with the components that it touches.
    E.g. "[lte][agw] Changeset" or "[orc8r][docker] ..."
-->


<!-- Enumerate changes you made and why you made them -->

## Test Plan
`go test builder_servicer_test.go`
<!--
    How did you test your change? How do you know it works?
    Add supporting screenshots, terminal pastes, etc. as necessary
-->

## Additional Information

- [ ] This change is backwards-breaking

<!--
    If this is a backwards-breaking change, document the upgrade instructions.
    All upgrade instructions for backwards-breaking changes will be aggregated
    in the next release's changelog so this is very important to fill out.
-->

xjtian commented on lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml:
```diff
@@ -1682,6 +1682,11 @@ definitions:
         minLength: 1
         maxLength: 4
+      sgi_management_iface_static_ip:
+        type: string
```
There is `format: ipv4` for IP addresses in swagger. Grep in the codebase for existing usages

pshelar committed:
[orc8r]: API: mconfig; sgi interface static IP address config.

Following API would allow operator configure a static
IP address for SGI management interface.

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>
